packages/documentation/docs/introduction.md

```diff
@@ -2,3 +2,62 @@
 id: introduction
 title: Introduction
 ---
+
+---
+
+## What is SPAship?
+# ![image alt text](image_0.png)
+SPAship is a platform for **deploying**, **integrating**, and **managing** single-page apps (SPAs).
+SPAship is an open source project. It is optimized to deploy and host SPAs in a very efficient and performant way. 
+
+### Confused ... :confused: Let me break it down for you :smiley: 
+SPAship as a product abstracts the underlying deployment infrastructure through a management portal(SPAship manager or CLI) and cloud-native deployment engine so that the developers can focus on developing SPAS rather than dealing with deployment/delivery and CI configurations. SPAship focuses on improving the deployment experience for the developers by integrating with existing CI and the deployment platform.
```

Comment:
```suggestion
SPAship as a product abstracts the underlying deployment infrastructure through a management portal (SPAship manager or CLI) and cloud-native deployment engine so that the developers can focus on developing SPAs rather than dealing with deployment/delivery and CI configurations. SPAship focuses on improving the deployment experience for developers by providing an easy to use CLI tool, an interactive user interface for web property management  and with  the ability to be integrated with existing CI based deployment platforms.
```